Sparkle/SUGuidedPackageInstaller.m

```diff
@@ -55,11 +55,11 @@ + (AuthorizationRef)authorizationForExecutable:(NSString*)executablePath
 		const char* executableFileSystemRepresentation = [executablePath fileSystemRepresentation];
 		
 		// Prepare a right allowing script to execute with privileges
-		AuthorizationItem right;
-		memset(&right,0,sizeof(right));
-		right.name = kAuthorizationRightExecute;
-		right.value = (void*) executableFileSystemRepresentation;
-		right.valueLength = strlen(executableFileSystemRepresentation);
+        AuthorizationItem right = {
```

Comment:
Ugh, it wasn't easy to make it backwards-compatible: http://stackoverflow.com/questions/31105433/how-to-use-objective-c-nonnull-in-a-backwards-compatible-way